packages/dates/src/date-time-utils/DateTimeUtils.ts

```diff
@@ -173,7 +173,11 @@ export function formatRelativeTime(
 
     if (formatted) {
       return formatted
+        .replace(/\sdays?/g, 'd')
```

Comment:
@doniyor2109 maybe we should format it manually instead of making one string and then replacing it with another, [DateTime#toRelative](https://moment.github.io/luxon/docs/class/src/datetime.js~DateTime.html#instance-method-toRelative) uses [Intl.RelativeTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat) under the hood (or it's ponyfill for outdated browsers), and if you check it's api you can find [Intl.RelativeTimeFormat#formatToParts](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat/formatToParts) which will allow us to format it manually. The problem is that `RelativeTimeFormat` uses time as a number, and we have to manually get it using [DateTime#diff](https://moment.github.io/luxon/docs/class/src/datetime.js~DateTime.html#instance-method-diff) or [DateTime#diffNow](https://moment.github.io/luxon/docs/class/src/datetime.js~DateTime.html#instance-method-diffNow)

Reply:
@doniyor2109 also checkout this [Duration#toFormat](https://moment.github.io/luxon/docs/class/src/duration.js~Duration.html#instance-method-toFormat)